src/main/scala/nl/knaw/dans/bag/v0/DansV0Bag.scala

```diff
@@ -477,13 +477,20 @@ class DansV0Bag private(private[v0] val locBag: LocBag) extends DansBag {
   override def addPayloadFile(src: File, pathInData: Path)
                              (implicit importOption: ImportOption): Try[DansV0Bag] = Try {
     val dest = data / pathInData.toString
-    mustNotExistInBag(dest)
-    mustBeChildOfBagData(dest)
+    if (dest == data && importOption == COPY) {
+      // allow to copy a directory to an empty data directory
+      if (!src.isDirectory || dest.list.nonEmpty)
```

Comment:
is it possible to inline this nested `if`? Or does that not work well with the `else` below?

Reply:
also, since this applies to the `COPY` option only, why don't you do this check in the match below?
same for the else branch, btw.

Reply:
Does the improved comment (only shown on files tab) explain the exception to the rule better?